logs_intake/socket/SendLogEntriesOverSocket.py

```diff
@@ -0,0 +1,104 @@
+# This code is provided as an example only.
+import json
+import time
+import socket
+import ssl
+
+ddApiKey = "<your_api_key>"
+
+# metadata: Additional metadata to send with the logs
+metadata = {
+    "ddsourcecategory": "aws",
+}
+
+host = "intake.logs.datadoghq.com"
+ssl_port = 10516
+
+DD_SOURCE = "ddsource"
+DD_CUSTOM_TAGS = "ddtags"
+
+metadata[DD_SOURCE] = "apigee"
+metadata[DD_CUSTOM_TAGS] = "level:info,x.api:identity-v1,env:dev,project:apigee"
+
+
+def lambda_handler(event):
```

Comment:
why are we calling this `lambda_handler`?

Reply:
Because that's what it was called when I borrowed the code - it came from our AWS logs lambda function.